src/resolve/resolvectl.c

```diff
@@ -1154,6 +1160,20 @@ static int show_statistics(int argc, char **argv, void *userdata) {
                            TABLE_FIELD, "Cache Misses",
                            TABLE_UINT64, n_cache_miss,
                            TABLE_EMPTY, TABLE_EMPTY,
+                           TABLE_STRING, "Serve Stale",
+                           TABLE_SET_COLOR, ansi_highlight(),
+                           TABLE_SET_ALIGN_PERCENT, 0,
+                           TABLE_EMPTY,
+                           TABLE_FIELD, "Total Timeouts",
+                           TABLE_SET_ALIGN_PERCENT, 100,
+                           TABLE_UINT64, n_total_timeouts,
+                           TABLE_FIELD, "Total Timeouts Protected",
+                           TABLE_UINT64, n_timeouts_served_stale,
+                           TABLE_FIELD, "Total Negative Responses",
```

Comment:
hmm, a negative response is also NXDOMAIN. which is not a case where we serve stale data. Maybe say "Failure Responses"?